### ISSUE TITLE
(URGENT) Fetching for relaton-data-iana failed due to rate limit

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-iana#14]
https://github.com/relaton/relaton-data-iana/runs/7789935144?check_suite_focus=true

```
2022-08-11T15:08:59.1125380Z Successfully installed relaton-cli-1.12.1
2022-08-11T15:08:59.1125724Z 119 gems installed
2022-08-11T15:08:59.1210556Z ##[group]Run rm -rf data
2022-08-11T15:08:59.1210835Z [36;1mrm -rf data[0m
2022-08-11T15:08:59.1211214Z [36;1mrelaton fetch-data iana-registries[0m
2022-08-11T15:08:59.1265065Z shell: /usr/bin/bash -e {0}
2022-08-11T15:08:59.1265338Z env:
2022-08-11T15:08:59.1266039Z   GITHUB_TOKEN: ***
2022-08-11T15:08:59.1266327Z ##[endgroup]
2022-08-11T15:09:03.6666536Z Rate limit is reached. Retrying in 30 sec.
2022-08-11T15:09:49.6516854Z Rate limit is reached. Retrying in 30 sec.
2022-08-11T15:10:47.5430776Z Rate limit is reached. Retrying in 30 sec.
2022-08-11T15:11:17.6410669Z Rate limit is reached. Retrying in 30 sec.
2022-08-11T15:11:47.7668481Z Rate limit is reached. Retrying in 30 sec.
2022-08-11T15:12:17.8942699Z /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-iana-1.12.4/lib/relaton_iana/data_fetcher.rb:58:in `fetch': You have exceeded a secondary rate limit. Please wait a few minutes before you try again. (StandardError)
2022-08-11T15:12:17.8944037Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-iana-1.12.4/lib/relaton_iana/data_fetcher.rb:63:in `fetch'
2022-08-11T15:12:17.8944897Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-iana-1.12.4/lib/relaton_iana/data_fetcher.rb:63:in `fetch'
2022-08-11T15:12:17.9036505Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-iana-1.12.4/lib/relaton_iana/data_fetcher.rb:63:in `fetch'
2022-08-11T15:12:17.9037180Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-iana-1.12.4/lib/relaton_iana/data_fetcher.rb:28:in `fetch'
2022-08-11T15:12:17.9037809Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-iana-1.12.4/lib/relaton_iana/processor.rb:32:in `fetch_data'
2022-08-11T15:12:17.9038441Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-cli-1.12.1/lib/relaton/cli/data_fetcher.rb:14:in `fetch'
2022-08-11T15:12:17.9039072Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-cli-1.12.1/lib/relaton/cli/command.rb:150:in `fetch_data'
2022-08-11T15:12:17.9039656Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
2022-08-11T15:12:17.9040257Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/thor-hollaback-0.2.1/lib/thor/hollaback.rb:71:in `block in run'
2022-08-11T15:12:17.9040853Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/hollaback-0.1.1/lib/hollaback/sequence.rb:34:in `call'
2022-08-11T15:12:17.9041421Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/thor-hollaback-0.2.1/lib/thor/hollaback.rb:71:in `run'
2022-08-11T15:12:17.9042005Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
2022-08-11T15:12:17.9042559Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
2022-08-11T15:12:17.9043096Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/thor-1.2.1/lib/thor/base.rb:485:in `start'
2022-08-11T15:12:17.9043645Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-cli-1.12.1/lib/relaton/cli.rb:41:in `start'
2022-08-11T15:12:17.9044230Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/lib/ruby/gems/3.1.0/gems/relaton-cli-1.12.1/exe/relaton:20:in `<top (required)>'
2022-08-11T15:12:17.9044619Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/bin/relaton:25:in `load'
2022-08-11T15:12:17.9044954Z 	from /opt/hostedtoolcache/Ruby/3.1.2/x64/bin/relaton:25:in `<main>'
2022-08-11T15:12:17.9949178Z Started at: 2022-08-11 15:09:03 +0000
2022-08-11T15:12:18.0414873Z ##[error]Process completed with exit code 1.
```

What's going on? Is this rate limiting from IANA.org, or from GitHub?

**Comment by andrew2net** [comment]
@ronaldtse it's already fixed in v 1.12.5 but we need to wait until the `ietf-tools/relaton-data-iana`'s GHA fetcher is run on the next schedule. I don't have permission to run it. The `relaton/relaton-data-iana` is updated successfully.

**Comment by ronaldtse** [comment]
Thanks!
